src/templates/page.js

```diff
@@ -80,6 +81,12 @@ export default ({ data, pageContent }) => {
   const { GATSBY_CARBON_ENV } = process.env;
   const isInternal = GATSBY_CARBON_ENV !== 'internal' && internal == true;
 
+  const classNames = classnames('page-content', {
+    'page-content--component': post.frontmatter.label === 'Component'
+  });
+
+  console.log(post.frontmatter.label);
```

Comment:
@alisonjoseph Was this a non-temporary debugging output?

Reply:
oops! will fix